### ISSUE TITLE
关于实验结果

### ISSUE THREAD

**Issue by zj223332** [Zhaoyang-Chu/HGRL-DTA#1]
作者，你好，您的代码我已经复现，但是在结果地方我有点问题，您的论文的结果与我直接划分后在训练集和测试集上运行的结果差不多，而我用您代码中在训练集上做的五折交叉验证的结果比您论文中的结果低，请问是什么清况，您的论文里的结果是在训练集上的五折交叉验证结果吗，还是整个数据集上做的五折交叉？ 我用的3080Ti跑的

**Comment by Zhaoyang-Chu** [comment]
Thanks for your question. As shown in Sec 5.3 of the paper, our experimental results are reported directly on the test set and do not involve 5-fold cross-validation. In our experiments, cross-validation is only conducted on the training data to select the optimal hyper-parameters for the model.

**Comment by lightwindcy** [comment]
> Thanks for your question. As shown in Sec 5.3 of the paper, our experimental results are reported directly on the test set and do not involve 5-fold cross-validation. In our experiments, cross-validation is only conducted on the training data to select the optimal hyper-parameters for the model.


I have read your code and paper, so the final results reported in the paper were retrained using both the training and validation sets, correct?

**Comment by Zhaoyang-Chu** [comment]
> > Thanks for your question. As shown in Sec 5.3 of the paper, our experimental results are reported directly on the test set and do not involve 5-fold cross-validation. In our experiments, cross-validation is only conducted on the training data to select the optimal hyper-parameters for the model.
> 
> I have read your code and paper, so the final results reported in the paper were retrained using both the training and validation sets, correct?

Yes. After selecting the optimal hyper-parameters for the model, we retrain the model using both the training and validation sets and then report the final results on the test set.
